pkg/provider/aws/resources/dynamodb.go

```diff
@@ -78,6 +78,38 @@ func (table *DynamodbTable) AttributeMap() (map[string]DynamodbTableAttribute, e
 	return attrs, merr.ErrOrNil()
 }
 
+type DynamodbTableCreateParams struct {
+	AppName string
+	Refs    []core.AnnotationKey
+	Name    string
+}
+
+func (table *DynamodbTable) Create(dag *core.ResourceGraph, params SecretVersionCreateParams) error {
+	table.Name = aws.DynamoDBTableSanitizer.Apply(fmt.Sprintf("%s-%s", params.AppName, params.Name))
```

Comment:
because we only ever have a single table i believe we only ever used the app name to generate the table, so would this cause us to fail if we had multiple tables? Or do we use env vars so we would just use multiple tables correctly?